common/src/main/resources/ts/utils/autocomplete/groupsSearch.ts

```diff
@@ -9,20 +10,31 @@ import {Group, GroupService} from "@common/services";
 
 export class GroupsSearch extends AutoCompleteUtils {
 
-    private groups: Array<Group>;
+    private groups: Array<Group | Grouping>;
     private selectedGroups: Array<{}>;
 
     private groupService: GroupService;
+    private groupingService: GroupingService;
 
     public group: string;
 
-    constructor(structureId: string, searchService: SearchService, groupService: GroupService) {
+    constructor(structureId: string, searchService: SearchService, groupService: GroupService, groupingService?: GroupingService) {
         super(structureId, searchService);
         this.groupService = groupService;
+        this.groupingService = groupingService;
     }
 
-    public getGroups() {
-        return this.groups;
+    //Return only group. Return group from grouping but not grouping
```

Comment:
J'aurais plus mis quelque chose du genre "Only returns group from grouping, but not the grouping itself" pour un peu plus de clarté